unified_planning/engines/factory.py

```diff
@@ -362,13 +364,14 @@ def _add_meta_engine(
 
     def _get_engine_class(
         self,
-        engine_kind: str,
+        engine_kind: "OperationMode",
```

Comment:
At this point I would rename this:
```suggestion
        operation_mode: "OperationMode",
```